modules/openapi-generator/src/main/resources/go-server/controller-api.mustache

```diff
@@ -543,7 +543,10 @@ func (c *{{classname}}Controller) {{nickname}}(w http.ResponseWriter, r *http.Re
 	{{paramName}}Param := getPointer({{paramName}}ParamValues[0])
 	{{/required}}
 	{{^required}}
-	{{paramName}}Param := getPointerOrNilIfEmpty(r.Header.Get("{{baseName}}"))
```

Comment:
Using `r.Header.Get` meant that we weren't referencing our previously defined `{{paramName}}ParamValues` variable so the compiler flagged the unused variable